src/Tizen.NUI/src/public/BaseComponents/VideoView.cs

```diff
@@ -447,6 +449,54 @@ public string ResourceUrl
             }
         }
 
+        /// <summary>
+        /// Enables auto rotation of the video based on the orientation of the video contents.
+        /// </summary>
+        /// <note> This feature is supported only when underlay is false. </note>
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        public bool IsAutoRotationEnabled
+        {
+            get
+            {
+                isAutoRotationEnabled = (bool)Interop.VideoView.IsAutoRotationEnabled(SwigCPtr);
+                if (NDalicPINVOKE.SWIGPendingException.Pending) throw NDalicPINVOKE.SWIGPendingException.Retrieve();
+                return isAutoRotationEnabled;
+            }
+            set
+            {
+                if (value != isAutoRotationEnabled)
+                {
```

Comment:
Ah, then should we use NUI private variable?
For example, if we can imagine the following case.

VideoView is underlay.
Set isAutoRotationEnabled = true
isAutoRotationEnable is true but DALi property is still false.
Set underlay = false.
Set isAutoRotationEnabled = true again.
Set is blocked by if statement and DALi property cannot be true.

In my opinion, we may not necessarily require a private variable here. While calling Interop might incur additional overhead, it's unlikely to significantly impact performance compared to managing complex logic internally.

Reply:
Oh, that's a good point. there is no need for private variables.